### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by akurtakov**

### CODE REVIEW

File: tycho-extras/tycho-p2-extras-plugin/src/test/java/org/eclipse/tycho/plugins/p2/extras/PublishFeaturesAndBundlesMojoTest.java [hunk 1]
```diff
@@ -94,10 +97,12 @@ private void initLegacySupport(List<MavenProject> projects, MavenProject current
         buildContext.setSession(session);
     }
 
-    private void generateContentToBePublished(File repositoryFolder) throws IOException {
+    private void generateContentToBePublished(String repositoryFolder) throws IOException {
```
Comment (mickaelistria) [inline tycho-extras/tycho-p2-extras-plugin/src/test/java/org/eclipse/tycho/plugins/p2/extras/PublishFeaturesAndBundlesMojoTest.java:100]: I don't think replacing a File by a String is a good idea; maybe by a Path?

**Review by mickaelistria — COMMENTED** [review]
Would it be possible to avoid the bulk formatting that applied to many files? It makes it almost impossible to review specific changes.

File: tycho-extras/tycho-p2-extras-plugin/src/test/java/org/eclipse/tycho/plugins/p2/extras/PublishFeaturesAndBundlesMojoTest.java (hunk 1 above)
Comment (akurtakov) [reply tycho-extras/tycho-p2-extras-plugin/src/test/java/org/eclipse/tycho/plugins/p2/extras/PublishFeaturesAndBundlesMojoTest.java]: Fixed with latest commit in the series.

**Review by mickaelistria — APPROVED** [review]
diisabling whtespace on the diff view helped at reviewing

**Pull request merged**